### PR TITLE
Add sort and actions to pfDataToolbar directive

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Added pfSimpleSort to the data toolbar
Added actions to the data toolbar
Updated toolbar styling based on latest Toolbar additions to Patternfly
    
Currently using a specific Patternfly commit as a dependency, this will need to be updated when patternfly/patternfly releases next and before we create a patternfly/angular-patternfly release

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -40,6 +40,6 @@
     "angular-route": "1.3.0 - 1.4.*",
     "angular-bootstrap": "0.13.x",
     "lodash": "3.x",
-    "patternfly": "2.2.0"
+    "patternfly": "https://github.com/patternfly/patternfly.git#41b81ba31dba9b451dc5b57b17a37b8d5d37d3af"
```
Comment (waldenraines) [inline bower.json]: Why this change?

Comment (jeff-phillips-18) [inline bower.json]: The current toolbar styling in Patternfly has not yet been released. As I put in the PR notes, we will need to update this once PF releases and before we release the next version of angular-patternfly

Comment (waldenraines) [inline bower.json]: Sorry I missed that in the notes.

**Comment by dtaylor113** [comment]
LGTM

File: src/views/toolbar/data-toolbar.html [hunk 2]
```diff
@@ -1,15 +1,51 @@
...
+                  ng-disabled="action.isDisabled === true">
+            {{action.name}}
+          </button>
+          <div class="dropdown btn-group" ng-if="config.actionsConfig.moreActions && config.actionsConfig.moreActions.length > 0">
+            <button type="button" class="btn btn-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              <span class="fa fa-ellipsis-v"></span>
+            </button>
+            <ul class="dropdown-menu">
+              <li ng-repeat="action in config.actionsConfig.moreActions"
+                  role="{{action.isSeparator === true ? 'separator' : ''}}"
```
Comment (waldenraines) [inline src/views/toolbar/data-toolbar.html]: Nitpick but shouldn't the role be something like `menuitem`?

**Comment by waldenraines** [comment]
LGTM pending rebase.

File: src/views/toolbar/data-toolbar.html (hunk 2 above)
Comment (jeff-phillips-18) [inline src/views/toolbar/data-toolbar.html]: fixed
